packages/frontend/src/stores/messagelist.ts

```diff
@@ -1195,3 +1190,20 @@ async function loadMessages(
 
   return await BackendRemote.rpc.getMessages(accountId, view)
 }
+
+// Custom hook to reactively access message list store state
+export const useMessageListStore = (messageListStore: MessageListStore) => {
+  const [, forceUpdate] = useReducer(x => x + 1, 0)
+
+  useEffect(() => {
+    const unsubscribe = messageListStore.subscribe(() => {
+      forceUpdate()
+    })
+    return unsubscribe
+  }, [messageListStore])
```

Comment:
I'm not sure when exactly `useEffect` runs. I'm a little afraid that this can result in some updates being missed. I would suggest to use the same approach that we use here:

https://github.com/deltachat/deltachat-desktop/blob/e76e5e6196f1f8143b8ef828fa8f8e49b2f5a40c/packages/frontend/src/stores/messagelist.ts#L144-L150

Or maybe to ensure synchronous execution of `subscribe`, e.g. with `useHasChanged2`.

But maybe it's just me not knowing React.